communities/asu-phoenix/community-assessment.md

```diff
@@ -0,0 +1,121 @@
+### Definition
+
+- Students at Arizona State University and the Phoenix Area interested in growing and becoming a part of the tech community. 
+
+### Size
+
+- 20000+ students enrolled at Arizona State University 
+    - 5,650 enrolled in Computing Program
+- 200-300 at large scale hackathons    
+
+- No target number, but want to ensure that all students, at such a large scale university, have a chance to learn more about technology and be a part of the community.
+
+- Recently, more hackathon groups have been arising and events that are more hands on have been able to see more than hundreds of students participaring! 
+    - The change is our school has become more accepting wih their new drive to be more innovative and have more of an entreprenial mindset. 
```

Comment:
This is great! Schools can change tactics quickly though, so I'd suggest finding out who is driving this change on the faculty, and integrating them in the work you're doing at the student level. Then you'll always have a champion on staff 💪